x-pack/plugins/security_solution/public/detection_engine/rule_response_actions/endpoint/utils.tsx

```diff
@@ -62,6 +62,48 @@ const useGetCommandText = (
           />
         ),
       };
+    case 'kill-process':
+      return {
+        title: (
```

Comment:
Do we already have `i18n` message defiend for some of these (ex. in the responder code)? if so, can we re-use them?

Reply:
There is some description in Response Console help section that we could use. However I took the above description from elastic docs. These will still undergo copy changes anyway. 
What do you think?